### PR TITLE
Internacionalizacao ingles para termos faltantes.

### PR DESCRIPTION
**Pull request by idgserpro**

### CODE REVIEW

File: src/brasil/gov/paginadestaque/configure.zcml [hunk 1]
```diff
@@ -23,4 +23,12 @@
   <!-- Temas Diazo -->
   <include file="themes.zcml" />
 
+  <utility
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/configure.zcml]: follow [ZCML style guide](http://docs.zope.org/zopetoolkit/codingstyle/zcml-style.html)

File: src/brasil/gov/paginadestaque/tests/test_setup.py [hunk 2]
```diff
@@ -56,13 +58,13 @@ def test_tiles_installed(self):
         self.assertIn(u'spacer', tiles)
 
     def test_types_not_searched(self):
-        site_properties = api.portal.get_tool('portal_properties')['site_properties']
+        site_properties = api.portal.get_tool('portal_properties')['site_properties']  # NOQA
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/tests/test_setup.py]: I prefer to fix this by ignoring E501 on code analysis configuration

File: src/brasil/gov/paginadestaque/tests/test_setup.py [hunk 3]
```diff
@@ -77,12 +79,78 @@ def test_tile_enabled(self):
         available_tiles = api.portal.get_registry_record(record)
         self.assertIn(u'spacer', available_tiles)
 
+
+class TestUpgrade(BaseTestCase):
+    """Ensure product upgrades work."""
+
+    def list_upgrades(self, source, destination):
+        upgradeSteps = listUpgradeSteps(self.st,
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/tests/test_setup.py]: follow conventions for [breaking lines](http://docs.plone.org/external/plone.api/docs/contribute/conventions.html#breaking-lines) here and all over the code

File: src/brasil/gov/paginadestaque/upgrades/v1001/handler.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+import logging
+
+
+def apply_profile(context):
+    """Atualiza perfil para versao 1001."""
+    logger = logging.getLogger(PROJECTNAME)
+    profile = 'profile-brasil.gov.paginadestaque.upgrades.v1001:default'
+    loadMigrationProfile(context, profile)
+    logger.info('Atualizado para versao 1001')
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/upgrades/v1001/handler.py]: provide a better description for the upgrade step and not just redundant information; AFAIK, this upgrade step adjusting the text on a content type.

File: src/brasil/gov/paginadestaque/tests/test_setup.py [hunk 5]
```diff
@@ -77,12 +79,78 @@ def test_tile_enabled(self):
...
+        else:
+            source = (source, )
+
+        step = [step for step in upgradeSteps
+                if (step[0]['dest'] == (destination,))
+                and (step[0]['source'] == source)]
+        return step
+
+    def execute_upgrade(self, source, destination):
+        # Setamos o profile para versao source
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/tests/test_setup.py]: comments in code in English, please

File: src/brasil/gov/paginadestaque/tests/test_setup.py [hunk 6]
```diff
@@ -77,12 +79,78 @@ def test_tile_enabled(self):
...
+    def test_to1001_execution(self):
+        self.execute_upgrade(u'1000', u'1001')
+        portal_types = api.portal.get_tool('portal_types')
+        sc_microsite = [type_info
+                        for type_info in portal_types.listTypeInfo()
+                        if type_info.id == 'sc.microsite'][0]
+        self.assertTrue(sc_microsite.title == 'Featured Home' and
+                        sc_microsite.description == 'Microsite for a campaign')
+
+    def test_ultimo_upgrade_igual_metadata_xml_filesystem(self):
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/tests/test_setup.py]: method names in English please

File: src/brasil/gov/paginadestaque/tests/test_setup.py [hunk 7]
```diff
@@ -77,12 +79,78 @@ def test_tile_enabled(self):
...
+        """
+        Testa se o número do último upgradeStep disponível é o mesmo do
```
Comment (hvelarde) [inline src/brasil/gov/paginadestaque/tests/test_setup.py]: docstrings in English, please

**Comment by idgserpro** [comment]
ZCML style guides and breaking lines issues were fixed. Other changes weren't made, check [brasil.gov.portlets justification for more information](https://github.com/plonegovbr/brasil.gov.portlets/pull/1#issuecomment-93834355).

Thanks for the review.

**Pull request merged**